source/chapters/advanced_topics.rst

```diff
@@ -105,7 +105,12 @@ work better with Mixxx (scratching), display a complex LED sequence, or even
 send messages to text displays on the controller.
 
 For more information, go to `<https://github.com/mixxxdj/mixxx/wiki/Midi-Scripting>`_
-and `<https://github.com/mixxxdj/mixxx/wiki/hid_mapping_format>`_.
+and `<https://github.com/mixxxdj/mixxx/wiki/hid_mapping_format>`_
```

Comment:
This is labeled as **specification for a future feature (not currently implemented)**
Please use the link https://github.com/mixxxdj/mixxx/wiki/Hid-Mapping instead
BTW: I don't think that this very old future feature spec is of any value in the Wiki.